packages/shared/src/components/modals/common.tsx

```diff
@@ -139,6 +143,7 @@ export const modals = {
   [LazyModal.ReputationPrivileges]: ReputationPrivilegesModal,
   [LazyModal.MarketingCta]: MarketingCtaModal,
   [LazyModal.UserSettings]: UserSettingsModal,
+  [LazyModal.Comment]: CommentModal,
```

Comment:
We could theoretically not include the comment modal in the lazy modal infra, since we are rendering it directly. I decided to keep it here in case we want to trigger it from other places manually. Let me know if you would instead prefer to remove it...

Reply:
We might have to not add it here completely since if we are going to need it, most likely we want the keyboard opened and this is only most likely used on mobile. But I'm open to keeping it here. Shouldn't hurt to be more prepared.

Reply:
OK, I'll remove it for now, since it's not needed to be lazily loaded.